src/components/App/App.scss

```diff
@@ -23,10 +23,19 @@ header {
   button {
     display: block;
     height: 70px;
-    width: 180px;
-    font-size: 1rem;
+    width: 220px;
```

Comment:
Maybe if were still having issues with the responsiveness we could try changing these to percentages